.github/workflows/releasedevhelm.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2025 Intel Corporation
```

Comment:
In this case, we usually use push action, and name the yaml as `push-xxx.yaml`.
https://github.com/opea-project/GenAIInfra/blob/d9e5ed02e4384894a49897fe4d56a34ab29fc302/.github/workflows/push-gmc-image-build.yaml#L7    
Should we also add a manually trigger action to release tag version helm charts? You can call it manual-xxx.yaml, like this https://github.com/opea-project/GenAIInfra/blob/main/.github/workflows/manual-image-build.yaml.